src/crypto/generichash/mod.rs

```diff
@@ -108,6 +108,11 @@ mod test {
     #[cfg(not(feature = "std"))]
     use prelude::*;
 
+    #[test]
+    fn test_crypto_generichash_state_alignment() {
+        assert_eq!(64, mem::align_of::<crypto_generichash_state>());
```

Comment:
Add comment why it should be 64 aligned

Reply:
Done, let me know if you have suggestions for anything you would like add or change.